temporal-serviceclient/src/main/java/io/temporal/serviceclient/WorkflowServiceStubs.java

```diff
@@ -61,7 +61,7 @@ static WorkflowServiceStubs newInstance(
 
   boolean isTerminated();
 
-  boolean awaitTermination(long timeout, TimeUnit unit) throws InterruptedException;
+  boolean awaitTermination(long timeout, TimeUnit unit);
```

Comment:
Seems like this return value has more meaning than it used to, would be nice to explain in docstring